### DISCUSSION TITLE
AttributeError: module 'pkgutil' has no attribute 'ImpImporter'. Did you mean: 'zipimporter'?

### DISCUSSION THREAD

**Discussion by annonymous-axe** [pypa/setuptools#4292] [Q&A]
When I run `pip install numpy == 1.24.2`.

I get the error message : 

`AttributeError: module 'pkgutil' has no attribute 'ImpImporter'. Did you mean: 'zipimporter'?`

Can anyone encounter the same issue before and how you solve it ?

**Comment by abravalheri** [comment]
The latest versions of setuptools has solved this problem a while ago. Só probably an older version of setuptools is being used.

If I had to guess I would say that there is a version cap for setuptools in `numpy == 1.24.2` that is not compatible with some latest version of Python (probably 3.12).

The easiest is to probably try using a more recent version of numpy.
